line-auth/functions/index.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for t`he specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+// Modules imports
+const functions = require('firebase-functions');
+const rp = require('request-promise');
+
+// Firebase Setup
+const admin = require('firebase-admin');
+const serviceAccount = require('./service-account.json');
+
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount),
+});
+
+// Load config file
+const config = require('./config.json');
```

Comment:
For configuration there is a special service built-in Cloud Functions for Firebase. See: 
https://github.com/firebase/functions-samples/blob/master/linkedin-auth/functions/index.js#L39-L40
and `firebase functions:config:set` commands in:
https://github.com/firebase/functions-samples/blob/master/linkedin-auth/README.md#setup-the-sample